package.json

```diff
@@ -8,50 +8,50 @@
     "url": "https://gitlab.com/antora/antora-ui-default.git"
   },
   "engines": {
-    "node": ">= 8.0.0"
+    "node": ">= 17.0.0"
   },
   "browserslist": [
     "last 2 versions"
   ],
   "devDependencies": {
     "@asciidoctor/core": "~2.2",
-    "autoprefixer": "~9.7",
+    "autoprefixer": "~10.4",
     "browser-pack-flat": "~3.4",
-    "browserify": "~16.5",
-    "cssnano": "~4.1",
+    "browserify": "~17.0",
+    "cssnano": "~5.1",
     "eslint": "~6.8",
     "eslint-config-standard": "~14.1",
     "eslint-plugin-import": "~2.20",
     "eslint-plugin-node": "~11.1",
     "eslint-plugin-promise": "~4.2",
     "eslint-plugin-standard": "~4.0",
-    "fancy-log": "~1.3",
+    "fancy-log": "~2.0",
     "font-awesome": "^4.7.0",
-    "fs-extra": "~8.1",
+    "fs-extra": "~10.0",
     "gulp": "~4.0",
     "gulp-concat": "~2.6",
     "gulp-connect": "~5.7",
     "gulp-eslint": "~6.0",
-    "gulp-imagemin": "~6.2",
-    "gulp-postcss": "~8.0",
+    "gulp-imagemin": "~8.0",
+    "gulp-postcss": "~9.0",
     "gulp-stylelint": "~13.0",
     "gulp-uglify": "~3.0",
-    "gulp-vinyl-zip": "~2.2",
+    "gulp-vinyl-zip": "~2.5",
     "handlebars": "~4.7",
     "highlight.js": "9.18.3",
-    "js-yaml": "~3.13",
+    "js-yaml": "~4.1",
     "merge-stream": "~2.0",
-    "postcss-calc": "~7.0",
-    "postcss-custom-properties": "~9.1",
-    "postcss-import": "~12.0",
-    "postcss-url": "~8.0",
-    "prettier-eslint": "~9.0",
+    "postcss-calc": "~8.2",
+    "postcss-custom-properties": "~12.1",
+    "postcss-import": "~14.1",
+    "postcss-url": "~10.1",
+    "prettier-eslint": "~13.0",
     "require-directory": "~2.1",
     "require-from-string": "~2.0",
     "stylelint": "~13.3",
```

Comment:
didn't update `stylelint` stuff since they have a bunch of new rules for css files
```
src/css/admonitionblock.css
 32:41   ✖  Expected modern color-function notation                 color-function-notation
 32:55   ✖  Expected "0.2" to be "20%"                              alpha-value-notation
 32:64   ✖  Expected modern color-function notation                 color-function-notation
 32:78   ✖  Expected "0.2" to be "20%"                              alpha-value-notation
 32:125  ✖  Expected line length to be no more than 120 characters  max-line-length
 59:25   ✖  Expected quotes around "url" function argument          function-url-quotes
 63:25   ✖  Expected quotes around "url" function argument          function-url-quotes
 67:25   ✖  Expected quotes around "url" function argument          function-url-quotes
 71:25   ✖  Expected quotes around "url" function argument          function-url-quotes
 75:25   ✖  Expected quotes around "url" function argument          function-url-quotes

src/css/base.css
 15:16  ✖  Expected custom property name to be kebab-case  custom-property-pattern

src/css/code-tabs.css
  6:3  ✖  Unexpected longhand value '1px 1px 0 1px'         shorthand-property-no-redundant-values
          instead of '1px 1px 0'
 12:1  ✖  Expected class selector to be kebab-case          selector-class-pattern
 23:1  ✖  Expected class selector to be kebab-case          selector-class-pattern
 24:3  ✖  Unexpected longhand value '0 0 0 0' instead of    shorthand-property-no-redundant-values
          '0'
 27:1  ✖  Expected class selector to be kebab-case          selector-class-pattern

src/css/doc.css
  14:16  ✖  Expected custom property name to be kebab-case  custom-property-pattern
  15:13  ✖  Expected custom property name to be kebab-case  custom-property-pattern
  16:16  ✖  Expected custom property name to be kebab-case  custom-property-pattern
 121:10  ✖  Expected custom property name to be kebab-case  custom-property-pattern
 129:10  ✖  Expected custom property name to be kebab-case  custom-property-pattern
 620:22  ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case
 629:17  ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case
 770:9   ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case
 858:21  ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case

src/css/header.css
   1:5   ✖  Expected class selector to be kebab-case        selector-class-pattern
 141:9   ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case
 199:28  ✖  Expected modern color-function notation         color-function-notation
 199:45  ✖  Expected "0.1" to be "10%"                      alpha-value-notation
 212:17  ✖  Expected custom property name to be kebab-case  custom-property-pattern
 266:17  ✖  Expected custom property name to be kebab-case  custom-property-pattern
 271:19  ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case
 301:17  ✖  Expected custom property name to be kebab-case  custom-property-pattern

src/css/highlight.css
  74:15  ✖  Expected class selector to be kebab-case  selector-class-pattern
  80:12  ✖  Expected class selector to be kebab-case  selector-class-pattern
  81:12  ✖  Expected class selector to be kebab-case  selector-class-pattern
  81:19  ✖  Expected class selector to be kebab-case  selector-class-pattern
  82:12  ✖  Expected class selector to be kebab-case  selector-class-pattern
 108:1   ✖  Expected class selector to be kebab-case  selector-class-pattern
 169:11  ✖  Expected class selector to be kebab-case  selector-class-pattern

src/css/main.css
 33:17  ✖  Expected custom property name to be kebab-case  custom-property-pattern

src/css/mode.css
 1:1   ✖  Expected class selector to be kebab-case        selector-class-pattern
 4:19  ✖  Expected quotes around "url" function argument  function-url-quotes
 7:21  ✖  Expected class selector to be kebab-case        selector-class-pattern
 8:19  ✖  Expected quotes around "url" function argument  function-url-quotes

src/css/nav.css
  49:13  ✖  Expected custom property name to be kebab-case  custom-property-pattern
  59:5   ✖  Expected class selector to be kebab-case        selector-class-pattern
  75:15  ✖  Expected modern color-function notation         color-function-notation
  75:29  ✖  Expected "0.5" to be "50%"                      alpha-value-notation
 133:31  ✖  Expected quotes around "url" function argument  function-url-quotes
 190:19  ✖  Expected quotes around "url" function argument  function-url-quotes
 200:3   ✖  Unexpected longhand value '0.5rem 0.75rem 0     shorthand-property-no-redundant-values
            0.75rem' instead of '0.5rem 0.75rem 0'
 253:17  ✖  Expected "currentColor" to be "currentcolor"    value-keyword-case

src/css/page-versions.css
 15:19   ✖  Expected quotes around "url" function argument          function-url-quotes
 31:133  ✖  Expected line length to be no more than 120 characters  max-line-length

src/css/print.css
 11:16  ✖  Expected custom property name to be kebab-case  custom-property-pattern

src/css/toolbar.css
 20:19  ✖  Expected quotes around "url" function argument  function-url-quotes
 38:25  ✖  Expected quotes around "url" function argument  function-url-quotes
 44:19  ✖  Expected quotes around "url" function argument  function-url-quotes
 52:25  ✖  Expected quotes around "url" function argument  function-url-quotes

src/css/typeface-roboto-mono.css
  8:9  ✖  Expected quotes around "url" function argument  function-url-quotes
  9:9  ✖  Expected quotes around "url" function argument  function-url-quotes
 19:9  ✖  Expected quotes around "url" function argument  function-url-quotes
 20:9  ✖  Expected quotes around "url" function argument  function-url-quotes

src/css/typeface-roboto.css
  2:16  ✖  Unexpected quotes around "Roboto"               font-family-name-quotes
  8:9   ✖  Expected quotes around "url" function argument  function-url-quotes
  9:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 13:16  ✖  Unexpected quotes around "Roboto"               font-family-name-quotes
 19:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 20:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 24:16  ✖  Unexpected quotes around "Roboto"               font-family-name-quotes
 30:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 31:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 35:16  ✖  Unexpected quotes around "Roboto"               font-family-name-quotes
 41:9   ✖  Expected quotes around "url" function argument  function-url-quotes
 42:9   ✖  Expected quotes around "url" function argument  function-url-quotes

src/css/vars-dark.css
  27:123  ✖  Expected line length to be no more than 120 characters  max-line-length
  29:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  30:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  45:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  52:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  82:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  85:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  91:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  92:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 140:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 147:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 149:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern

src/css/vars.css
  20:123  ✖  Expected line length to be no more than 120 characters  max-line-length
  22:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  23:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  38:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  45:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  75:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  79:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  85:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
  86:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 134:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 141:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
 143:3    ✖  Expected custom property name to be kebab-case          custom-property-pattern
```